.gitignore

```diff
@@ -21,4 +21,20 @@ __pycache__/
 
 #idea/pycharm
 .idea/
-.tox
\ No newline at end of file
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
```

Comment:
we're not using this guy, are we?

Reply:
You are right, we are not using this, so, since this is already merged,  I made the correction in (#1915)

Thanks @AndreMiras!!